dataparser.go

```diff
@@ -135,13 +135,20 @@ func (p *nullableParser) Parse(s io.RuneScanner) (driver.Value, error) {
 
 		isNotString := false
 		for {
-			r, _, err := s.ReadRune()
+			r, size, err := s.ReadRune()
+			if size == 0 {
```

Comment:
are you sure we need to check the size first? 
Because in this case we never get to check the error.
This is a typical return statement from ReadRune
```
return nil, 0, err 
```

Reply:
I know, but if we try to call ReadRune() on empty string - we'll always get error ("EOF"), because we fall in first if statement

```	
if r.i >= int64(len(r.s)) {
		r.prevRune = -1
		return 0, 0, io.EOF
	}
```

Reply:
consider
```
if err == io.EOF { 
   break
}
```

Reply:
Looks good, use it